docs/security.adoc

```diff
@@ -252,17 +248,17 @@ You should see:
 [source,none]
 ----
 HTTP/1.1 200 OK
-Connection: keep-alive
+Content-Length: 2256
 Content-Type: text/plain;charset=UTF-8
-Content-Length: 2272
-Date: Tue, 16 Jul 2019 00:14:54 GMT
+Set-Cookie: 2a1b392100b8b2cb3705c68f4ecbaf66=1b3560b80b9fad566e105aff1f31f880; path=/; HttpOnly
+Cache-control: private
 
-hello admin, isSecure: false, authScheme: MP-JWT, hasJWT: true
-Injected issuer: ["{{KEYCLOAK_URL}}/auth/realms/quarkus"]
+hello admin, isSecure: false, authScheme: Bearer, hasJWT: true
+Injected issuer: ["http://keycloak-codeready.apps.cluster-testing-4b1e.testing-4b1e.sandbox404.opentlc.com/auth/realms/quarkus"]
```

Comment:
The use of ``{{KEYCLOAK_URL}}`` should be preserved here so output matches each individual user.

Reply:
Fixed